controllers/topicPage/TopicPage.js

```diff
@@ -108,7 +116,8 @@ class TopicPage {
         limit,
         offset,
         userId: req.userId,
-        anonymousUserId: myAnonymousUser?.user_id
+        anonymousUserId: myAnonymousUser?.user_id,
+        isBlurredPost
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isBlurredPost` is added to the payload. Make sure that wherever this payload is used, the `isBlurredPost` property is handled correctly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->